docs/core-concepts.md

```diff
@@ -129,28 +100,25 @@ set will be able to opt-in to become a validator for other zones.
 
 ## Fraud Proofs
 
-The Rollkit chains will have no consensus rules. All valid transactions will
-be accepted by block creators. Fraud proofs will be the only way to detect
-and rollback malicious state transitions.
+The Rollkit chains will inherit the consensus of the consensus layer when
+it comes to ordering rollup blocks. All valid rollup blocks on the DA layer
+will be accepted by nodes. In the case of a pessimistic rollup, you can
+run a full node for the rollup for fraud proofs. In a zk-rollup, there will
```

Comment:
Pessimistic rollups are rollups where each node is full - there is no need for fraud proofs as everyone executes all transactions.
Optimistic rollups require fraud proofs. Full nodes are needed to create fraud proofs.

Reply:
copy that. thank you for this review Tomasz. Will implement in PR for core concepts